### PR TITLE
Feat/django 42 compatible

### PR DESCRIPTION
**Pull request by vipulnarang95**
Django 4.2 support added
Dropped support for Django<3.2, python<3.8
Added support for python 3.10

### CODE REVIEW

**Comment by vipulnarang95** [comment]
@kinkerl Can you please review this?

**Comment by joshyu** [comment]
@FinalAngel  / @kinkerl / @jonathan-s ,
Can you please help to review this PR and merge if no error?

Cheers.

File: tests/requirements/requirements_base.txt [hunk 1]
```diff
@@ -12,4 +16,4 @@ mock
 # Unreleased django-cms 4.0 compatible packages
 https://github.com/django-cms/django-cms/tarball/release/4.0.1.x#egg=django-cms
 https://github.com/django-cms/djangocms-versioning/tarball/support/django-cms-4.0.x#egg=djangocms-versioning
-https://github.com/Aiky30/djangocms-moderation/tarball/master#egg=djangocms-moderation
+https://github.com/FidelityInternational/djangocms-moderation/tarball/feature/django-42-compat#egg=djangocms-moderation
```
Comment (joshyu) [inline tests/requirements/requirements_base.txt:19]: @vipulnarang95 , 
please use `https://github.com/django-cms/djangocms-moderation/tarball/master#egg=djangocms-moderation` , as **Fabian** has already finished Django 4.2 support on it and merged in master branch.

**Comment by fscherf** [comment]
Hi @vipulnarang95 and @joshyu!
@kinkerl is on vacation, so I will review this PR as requested.

I looked over the changes and tested the PR on my machine. So far everything looks ok to me!

**Review by fscherf — APPROVED** [review]

File: tests/requirements/requirements_base.txt (hunk 1 above)
Comment (vipulnarang95) [reply tests/requirements/requirements_base.txt]: done

**Pull request merged**